src/window.rs

```diff
@@ -218,26 +206,19 @@ unsafe impl Send for SendSyncRawWindowHandle {}
 unsafe impl Sync for SendSyncRawWindowHandle {}
 
 impl WindowAttributes {
-    /// Get the parent window stored on the attributes.
-    #[cfg(feature = "rwh_06")]
-    pub fn parent_window(&self) -> Option<&rwh_06::RawWindowHandle> {
-        self.parent_window.as_ref().map(|handle| &handle.0)
-    }
-}
-
-impl WindowBuilder {
     /// Initializes a new builder with default values.
     #[inline]
-    #[deprecated = "use `Window::builder` instead"]
+    #[deprecated = "use `Window::default_attributes` instead"]
```

Comment:
Isn't the point to suggest Window type to avoid import in the first place? Otherwise I'd rather remove the thing all together and say the users to just `WindowAttributes::default` or something.

Reply:
Yeah, I'm fine with just suggesting to use `Window::default_attributes`